packages/ses/src/lockdown.js

```diff
@@ -408,6 +382,12 @@ export const repairIntrinsics = (options = {}) => {
     markVirtualizedNativeFunction,
   });
 
+  if (legacyHermesTaming === 'unsafe') {
+    globalThis.testCompartmentHooks = undefined;
+    // @ts-ignore Compartment does exist on globalThis
+    delete globalThis.Compartment;
```

Comment:
getting rid of Compartment makes sense for Hermes for now, but eventually we will want the `compartmentInstance.globalThis` to work correctly, so I'd rather make the Compartment's code eval capabilities go.

Either remove or, I think preferably, replace with a function that throws a descriptive error, the following:
- `c.evaluate` instance function
- Compartment constructor accepting importHook and related
I think that could be achieved by checking the option and throwing an error in make safe evaluator somewhere.